src/psyclone/psyGen.py

```diff
@@ -4327,12 +4356,12 @@ def search_subroutine(nodelist, searchname):
 
     def process_declarations(self, parent, nodes):
         '''
-        Transform the fparser2 AST declarations to symbols into the PSyIRe
+        Transform the fparser2 AST declarations to symbols into the PSyIR
```

Comment:
"Transform the variable declarations in the fparser2 parser tree into symbols in the ...."

Reply:
Done